Lib/test/test_sndhdr.py

```diff
@@ -25,6 +26,30 @@ def test_data(self):
             self.assertEqual(what.nframes, expected[3])
             self.assertEqual(what.sampwidth, expected[4])
 
+    def test_badinputs(self):
```

Comment:
Since all of these are expected to fail and return `None`, I'd suggest simplifying those loops to:

```python
for filename in ['input1.bad', 'input2.bad']:
    ...
    self.assertEqual(whathdr, None)
```

Reply:
that's fine by me! I'll make the change